bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java

```diff
@@ -462,4 +487,9 @@ private WebElement getClickableParsys() {
       return xPositionDiff + yPositionDiff <= parsysOffset;
     };
   }
+
+  private void waitForComponentRemoved(WebElement webElement) {
```

Comment:
waitForComponentRemoval or waitForComponentToBeRemoved to be more grammatically correct?